build.fsx

```diff
@@ -164,7 +164,7 @@ Target "NuGet" <| fun () ->
 // Generate the documentation
 
 Target "GenerateDocs" <| fun () ->
-    executeFSIWithArgs "docs/tools" "generate.fsx" ["--define:RELEASE"] []
+    executeFSIWithArgs "docs/output" "generate.fsx" ["--define:RELEASE"] []
```

Comment:
Sorry about the confusion - I've been more concerned with designing the F# API than generating the documentation for it as I'm going along. Please revert this change when you get a chance.

You can see the same [`"GenerateDocs"` target in FSharp.Data's build script](https://github.com/fsharp/FSharp.Data/blob/2.3.0/build.fsx#L191) and [what an example`generate.fsx` looks like](https://github.com/fsharp/FSharp.Data/blob/2.3.0/docs/tools/generate.fsx).